src/validation/PhoneValidator.java

```diff
@@ -0,0 +1,37 @@
+package validation;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class PhoneValidator {
+	
+	private Map<String, Integer> countryPrefix = new HashMap<>();
+	
+	public PhoneValidator() {
+		countryPrefix.put("+49", 9);
+		countryPrefix.put("+370", 8);
+	}
+
+	public boolean hasOnlyNumbers(String string) {
+		for (int i=0; i < string.length(); i++) {
+			if (!Character.isDigit(string.charAt(i))) {
+				return false;
+			}
+		}
+		return true;
+	}
+
+	public boolean beginsWithEight(String string) {
+		if (string.charAt(0) == 56) return true;

```

Comment:
Try to avoid magic numbers - 56 is a bit cryptic what it means. I get it's ACII for 8, but that's still poor readability.